app/src/ui/app.tsx

```diff
@@ -1456,13 +1458,38 @@ export class App extends React.Component<IAppProps, IAppState> {
           return null
         }
 
+        // upstream github repo, if there is one
+        const upstreamGhRepo: GitHubRepository | null =
+          enableForkyCreateBranchUI() &&
```

Comment:
if this is false, then the following values will be null, resulting in the usual version of the create branch dialog for all repos, including forks.